notebook/static/base/js/i18nload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Plugin to load a single locale.
+ */
+define([
+	"require",
+	"module",
+	 // These are only here so that the optimizer knows which ones we MIGHT load.
+	 // We will actually only load the ones we need. There should be one entry
+	 // here for each language you want to support.
+	 // For example, for German....
+	 // "json!base/../../i18n/de/LC_MESSAGES/nbjs.json"

```

Comment:
The optimizer doesn't look at commented values.  You will need to comment them back in in order to support a particular language.  The answer to your 2nd question is, nothing stops them, as they will all get loaded if there are a lot of them.   The current implementation is not optimal, and I understand that.   However, since we don't have real translations yet, I don't want this to be a showstopper issue for getting the merge done.  I can work on improving the plugin so that it is more robust.   One approach that I have used in the past is to allow the set of translations you want to load to be configurable via require.config(), and then have the plugin read that.